integration/simple-prototype-defaults/simple.ts

```diff
@@ -1004,42 +1011,49 @@ export const SimpleWithMap = {
       const tag = reader.uint32();
       switch (tag >>> 3) {
         case 1:
+          message.entitiesById ||= [];
```

Comment:
Shouldn't all the properties under this `SimpleWithMap` be instantiated with an empty object, i.e. `{}` instead of array?